docs/api.md

```diff
@@ -312,8 +312,6 @@ for `code` and `errno` are:
   A backend service request failed.
 * `code: 500, errno: 998`:
   An internal validation check failed.
-* `code: 500, errno: 999`:
-  Unspecified error
```

Comment:
Evidently I angered our capricious docs script again somehow, I'll dig into that separately.